app/adapters/alma_adapter/alma_item.rb

```diff
@@ -203,26 +206,50 @@ def item_type
       item_data.dig("policy", "value")
     end
 
-    def status_label(marc_holding)
-      value = marc_holding["availability"]
-      if value.nil?
-        # TODO: This will need to be figure out later on.
-        #
-        #   It possible that we will never hit this case because Request might not need to
-        #   find out the availability for holding that meets this criteria but if it does
-        #   then the current code will handle it.
-        #
-        #   The issue here is that the holding information that comes for these records
-        #   does not have a holding for the holding_id that we are working with. Instead it has
-        #   a single holding that does NOT have an ID. This seems to be an issue only for
-        #   eresources.
-        #
-        #   For an example see: http://localhost:3000/bibliographic/9919392043506421/holdings/22105104420006421/availability.json
-        #
-        #   For now use the data in the item.
-        value = item_data["base_status"]["desc"]
-      end
-      value
+    def calculate_status
+      return status_from_work_order_type if item_data.dig("work_order_type", "value").present?
+      return status_from_process_type if item_data.dig("process_type", "value").present?
+      status_from_base_status
+    end
+
+    def status_from_work_order_type
+      value = item_data["work_order_type"]["value"]
+      desc = item_data["work_order_type"]["desc"]
+
+      # Source for values: https://developers.exlibrisgroup.com/alma/apis/docs/xsd/rest_item.xsd/
+      # and https://api-na.hosted.exlibrisgroup.com/almaws/v1/conf/departments?apikey=YOUR-KEY&format=json
+      code = if value == "Bind" || value == "Pres" || value == "CDL"
+               "Not Available"
+             else
+               # "COURSE" or "PHYSICAL_TO_DIGITIZATION"
+               "Available"
+             end
+      { code: code, label: desc, source: "work_order" }
+    end
+
+    def status_from_process_type
+      value = item_data.dig("process_type", "value")
+      desc = item_data.dig("process_type", "desc")
+
+      # Source for values: https://developers.exlibrisgroup.com/alma/apis/docs/xsd/rest_item.xsd/
+      code = if value == "ACQ"
```

Comment:
Knowing the value of the process type would be useful. Currently we have "special" behaviors for titles that are "On Order" which is what the process type "ACQ" would map to.

Reply:
Good to know. I've added a `process_type` property to the response.